### DISCUSSION TITLE
Cannot create an acme-challenge on the .well-known directory.

### DISCUSSION THREAD

**Discussion by ErginDapaj** [vercel/community#5289] [Help]
### Summary

Alright so yes i do have directories enabled and i can access the .well-known directory where i can see all the files and folders. But once i try to access a folder like acme-challenge it gives me a 404.
error	
code	"not_found"
message	"Not Found"

Now i tried this on my nextjs app then on a simple html site and i got the same results...

### Example

https://ergin.al/.well-known

### Steps to Reproduce

Add a folder with content to a .well-known directory, and you wont be able to access it.

**Comment by YoannBuzenet** [comment]
Hello,
I have the problem too.

Here the certbot output on my nextjs app. I changed the domain to xxx.xxx to not disclose mine.

```  Domain: xxx.xxx
  Type:   unauthorized
  Detail: 82.66.128.239: Invalid response from http://xxx.xxx.well-known/acme-challenge/QoJw47Wdj7Gq-LWW19NqI-h2FsWAOqwyQeDDk1P84Nk: 404

Hint: The Certificate Authority failed to download the challenge files from the temporary standalone webserver started by Certbot on port 80. Ensure that the listed domains point to this machine and that it can accept inbound connections from the internet.

Cleaning up challenges
```
The app gives indeed a 404 when trying the URL. Is it a normal behaviour ? Does anyone knows a way to avoid it ?
